WORKSPACE

```diff
@@ -50,6 +54,18 @@ npm_import(
     package_version = "8.4.0",
 )
 
+translate_pnpm_lock(
```

Comment:
suppose we'll need to make these available to user workspaces via a bzl file since they'll need to be initialized downstream

Reply:
we don't run the e2e workspaces yet in CI so you have to do it manually, but you can tease out the error by running
`bazel build ...` in `/Users/greg/aspect/rules/rules_js/e2e/node_package`

```
$ bazel build ...
ERROR: /Users/greg/aspect/rules/rules_js/e2e/node_package/BUILD.bazel:3:13: error loading package '@aspect_rules_js//js/private/lifecycle': Unable to find package for @aspect_rules_js_lifecycle_npm_deps//:package.bzl: The repository '@aspect_rules_js_lifecycle_npm_deps' could not be resolved: Repository '@aspect_rules_js_lifecycle_npm_deps' is not defined. and referenced by '//:npm__at_aspect-test_qar_4.0.0_postinstall'
ERROR: Analysis of target '//:npm__at_aspect-test_qar_4.0.0_postinstall' failed; build aborted: Analysis failed
```

Reply:
Not sure how to handle the `register_yq_toolchains`. Do we do this automatically for users in a setup macro or do we ask users to register explicitly. @alexeagle wtyt?